### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by animeshgarai09**
Shop page done
[https://deploy-preview-5--hungry-hawking-2a29f9.netlify.app/shop.html](url)

### CODE REVIEW

**Review by MayankKumar10 — APPROVED** [review]
LGTM you can merge it

**Review by itsmebipul00 — APPROVED** [review]

File: styles/global.scss [hunk 1]
```diff
@@ -69,6 +69,10 @@ body {
         align-items: center;
         padding: 0.6rem 4rem;
     }
+    .sub__wrapper {
```
Comment (manojsarna) [inline styles/global.scss:72]: can use px if values are less than 10 px

File: styles/product.scss [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    flex: 1;
+    .small {
+        .card__image {
+            min-width: 13rem;
+            max-height: 12.75rem;
+        }
+        .card__name {
+            font-size: 0.9rem;
+        }
+        .card__content p {
```
Comment (manojsarna) [inline styles/product.scss:29]: great calculations

File: styles/product.scss [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            -webkit-line-clamp: 2;
+        }
+    }
+}
+.side {
+    padding: 1.3rem;
+    box-shadow: var(--box-shadow-1);
+    border-radius: 0.8rem;
+    border: 1px solid var(--border-color-3);
```
Comment (manojsarna) [inline styles/product.scss:38]: can use px

File: styles/product.scss [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        justify-content: space-between;
+        align-items: center;
+        margin-bottom: 0.6rem;
+    }
+    .heading {
+        display: inline-block;
+        font-size: 0.94rem;
+        font-weight: 600;
+        margin: 0.7rem 0;
+    }
```
Comment (manojsarna) [inline styles/product.scss:56]: can use px

**Review by manojsarna — APPROVED** [review]
overall code is great. Thoughful coding

Looks Good to merge

**Pull request merged**